components/src/molecules/select/Select.stories.js

```diff
@@ -0,0 +1,89 @@
+import { storiesOf } from '@storybook/vue'
+import { action } from '@storybook/addon-actions'
+
+import countries from '../../../mocks/countries.json'
+
+import ASelect from './Select.vue'
+
+storiesOf('Molecules/Select', module)
+  .addParameters({ info: true })
+  .add('Default', () => ({
```

Comment:
Please format the same way as radio/checkbox

Reply:
I'm not a huge fan of this formatting, but ok, done :P